frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -38,11 +41,77 @@ export const Dashboard = () => {
       setGroups(res.data.data)
     })
   }, [])
+  // (a,b) = -1 if a before b, 1 if a after b, 0 if equal
+  function sorted(groups: CourseInfo[], menuValue: string) {
+    if (menuValue === 'newly-match-first') {
+      return [...groups].sort((a, _) => {
+        //-1 if a newly matchable and b isn't
+        if (a.lastGroupNumber === 0 && a.unmatched.length > 1) {
+          return -1
+        } else return 1
+      })
+    } else if (menuValue === 'unmatch-first') {
+      return [...groups].sort((a, _) => {
+        //-1 if a unmatchable and b isn't
+        if (a.lastGroupNumber === 0 && a.unmatched.length === 1) {
+          return -1
+        } else return 1
+      })
+    } else if (menuValue === 'match-first') {
+      return [...groups].sort((a, _) => {
+        //-1 if a matchable and b isn't
+        if (a.lastGroupNumber > 0 && a.unmatched.length > 0) {
+          return -1
+        } else return 1
+      })
+    } else if (menuValue === 'classesa-z') {
+      const sorted = [...groups].sort((a, b) => {
+        return a.names[0].localeCompare(b.names[0], undefined, {
+          numeric: true,
+        })
+      })
+      return sorted
+    } else if (menuValue === 'classesz-a') {
+      const sorted = [...groups].sort((a, b) => {
+        return b.names[0].localeCompare(a.names[0], undefined, {
+          numeric: true,
+        })
+      })
+      return sorted
+    } else return groups
+  }
+  const handleChange = (event: SelectChangeEvent) => {
+    setStatus(event.target.value)
+    setGroups(sorted(groups, event.target.value))
+  }
 
   return (
     <StyledContainer>
       <StyledHeaderMenu>
         <LogoImg />
+        <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+          <Box
+            sx={{ fontWeight: 'bold', color: '#5C5B6A', padding: 1, margin: 1 }}
```

Comment:
@mluo24 knows more than I do, but I believe you can reference this color with `color: 'essentials.75'` instead of hard coding it, since `sx` has access to the theme